src/js/util/ui.js

```diff
@@ -111,7 +111,7 @@ define(function (require, exports) {
     var openURL = function (url, event, name) {
         adapter.openURLInDefaultBrowser(url);
 
-        headlights.logEvent("UserInterface", "introduction", name);
```

Comment:
libraries links (e.g. share library/element) are also using `openURL`, so the subcategory should not be `introduction` here. Maybe a better way is putting the `logEvent` in the first launch element.